components/brave_vpn/brave_vpn_service.cc

```diff
@@ -1202,41 +1202,8 @@ void BraveVpnService::OnGetResponse(
     int status,
     const std::string& body,
     const base::flat_map<std::string, std::string>& headers) {
-  std::string json_response;
```

Comment:
With this code gone, maybe we can have a comment in `BraveVpnService::OAuthRequest` with something like:
```
// NOTE: api_request_helper_ uses JsonSanitizer to sanitize input made with requests
```

Reply:
Added 👍🏼